app/controllers/ctc/questions/confirm_payment_controller.rb

```diff
@@ -0,0 +1,29 @@
+module Ctc
+  module Questions
+    class ConfirmPaymentController < QuestionsController
+      include AuthenticatedCtcClientConcern
+
+      layout "intake"
+
+      def edit
+        @ctc_amount = ChildTaxCreditCalculator.total_advance_payment_2021(current_intake.dependents)
+        @rrc_amount = current_intake.tax_return(2020)&.claimed_recovery_rebate_credit || 0
```

Comment:
since we're calling current_intake.tax_return(2020) multiple times here, I think you could improve readability by defining tax_return once and calling methods on it.

There should be no situation here where a client does _not_ have a 2020 tax return, and claimed_recovery_rebate_credit should respond with 0 when they aren't claiming credit, so I'd recommend removing the &. and || 0 to simplify the logic a bit.

There are cases I've seen where test data (and real data) can cause the calculation to error out, but I actually don't think that either of the conditions above would help in that situation (we need to fix it separately).